packages/govuk-frontend/src/govuk/components/accordion/accordion.puppeteer.test.js

```diff
@@ -726,7 +726,8 @@ describe('/components/accordion', () => {
           ).rejects.toMatchObject({
             cause: {
               name: 'SupportError',
-              message: 'GOV.UK Frontend is not supported in this browser'
+              message:
+                'GOV.UK Frontend is missing `<body class="govuk-frontend-supported">` via inline `<script>`'
```

Comment:
We should get feedback from @claireashworth too but I'm changing my own suggestion 🙈

```suggestion
                'GOV.UK Frontend initialised without `<body class="govuk-frontend-supported">` from template `<script>` snippet'
```